### PR TITLE
feat: fix strip

### PR DESCRIPTION
**Pull request by o0OooO**
## Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: aws_extension/sagemaker_ui_tab.py [hunk 1]
```diff
@@ -175,10 +175,10 @@ def ui_user_settings_tab():
                                              "refresh_username")
         gr.HTML('<b>Password</b>')
         with gr.Row():
-            password_textbox = gr.Textbox(value=get_variable_from_json('password'), type='password', interactive=True,
+            password_textbox = gr.Textbox(type='password', interactive=True,
                                           placeholder='Please enter your password', show_label=False)
-            modules.ui.create_refresh_button(password_textbox, lambda: get_variable_from_json('password'),
-                                             lambda: {"value": get_variable_from_json('password')},
+            modules.ui.create_refresh_button(password_textbox, lambda: get_variable_from_json('username'),
```
Comment (alvindaiyan) [inline aws_extension/sagemaker_ui_tab.py]: why username to a password textbox?

**Review by alvindaiyan — APPROVED** [review]
lgtm, goooo

**Comment by NingLu** [comment]
We identified new issues on unchanged lines of code. Navigate to the Amazon CodeGuru Reviewer console to view the recommendations to fix them.

File: aws_extension/sagemaker_ui.py [hunk 2]
```diff
@@ -785,12 +785,17 @@ def process_result_by_inference_id(inference_id):
 
     resp = get_inference_job(inference_id)
     if resp is None:
-        return image_list, info_text, plaintext_to_html(infotexts)
+        logger.info(f"get_inference_job resp is null")
+        return image_list, info_text, plaintext_to_html(infotexts), infotexts
```
Comment (NingLu) [inline aws_extension/sagemaker_ui.py:789]: Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears you are unpacking more than three variables while using a function that returns more than one variable. Using a large number of return values is prone to errors. We recommend that you return a small class or `namedtuple` instance instead.

[Learn more](https://docs.python.org/3/library/typing.html#typing.NamedTuple)

Similar issue at line numbers 798, 801, 806, 829, 832, and 834.

**Pull request merged**